### PR TITLE
Network-logger changes to support DFL-2491: Resource inspector improvements - phase 1

### PR DESCRIPTION
**Pull request by danfooo**

### CODE REVIEW

File: src/network/network_service.js [hunk 1]
```diff
@@ -2,6 +2,24 @@
 
 cls.NetworkLogger = function()
 {
+  this._filter_entries_by_list = function(ids, entry)
+  {
+    return ids.contains(entry.id);
+  }
```
Comment (hzr) [inline src/network/network_service.js]: Missing semicolon.

File: src/network/network_service.js [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+  {
+    // ids is an optional array of entry ids.
+    var ctx = this.get_logger_context();
+    var entries = ctx && ctx.get_entries() || [];
+    if (ids)
+    {
+      var filter_bound = this._filter_entries_by_list.bind(this, ids);
+      entries = entries.filter(filter_bound);
+    }
+    return entries.map(function(entry) { return new cls.ResourceInfo(entry);} );
```
Comment (hzr) [inline src/network/network_service.js]: Space before the end bracket of the function body.

File: src/network/network_view.js [hunk 3]
```diff
@@ -165,7 +168,11 @@ cls.NetworkLogView = function(id, name, container_class, html, default_handler,
     if (this.selected && ctx.get_entry_from_filtered(this.selected))
     {
       if (this._overlay.is_active)
-        this._overlay.update();
+      {
+        if (!this._last_updated_entry || this._last_updated_entry == this.selected)
+          this._overlay.update();
+
```
Comment (hzr) [inline src/network/network_view.js]: Extra newline.

File: src/network/network_view.js [hunk 4]
```diff
@@ -590,15 +607,24 @@ cls.NetworkLogView = function(id, name, container_class, html, default_handler,
...
       this.update();
   }.bind(this);
 
+  this._on_resource_update_bound = function(message)
+  {
+    if (message.is_paused)
+      return;
+
+    if (message.id)
+    {
```
Comment (hzr) [inline src/network/network_view.js]: Might skip braces here.

**Comment by p01** [comment]
Looks good to me.

**Pull request merged**